deeplearning-models/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+set( TARGET deeplearning-models )
+set( TARGET_ROOT ${topdir}/deeplearning-models )
+
+message( STATUS "========== Configuring ${TARGET} ==========" )
+
+def_vars()
+
+list( APPEND SOURCES
+   resampler.pt
+   modelcard-schema.json
+)
+
+foreach( source ${SOURCES} )
+   set( src "${_SRCDIR}/${source}" )
+   if( CMAKE_SYSTEM_NAME MATCHES "Darwin|Windows" )
+      set( dst "${_DEST}/${TARGET}/${source}" )
+   else()
+      set( dst "${CMAKE_BINARY_DIR}/lib/${TARGET}/${source}" )
+   endif()
+
+   add_custom_command(
+      DEPENDS
+         "${src}"
+      COMMAND
+         ${CMAKE_COMMAND} -E make_directory "${_DEST}/${TARGET}"
+      COMMAND
+         ${CMAKE_COMMAND} -E copy "${src}" "${dst}"
+      OUTPUT
+         "${dst}"
+   )
+
+   list( APPEND OUTPUTS "${dst}" )
```

Comment:
Similarly, precede the loop with the command `set( OUTPUTS )` to be sure that variable is empty before you append to it.

Reply:
resolved in 7e642c2d8da630073e45fe99d4e9687131e97a22